res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1624,15 +1624,18 @@ NumarkMixtrack3.SamplerButton = function(channel, control, value, status, group)
                 engine.setValue(group, "eject", 1);
                 engine.setValue(group, "eject", 0);
             } else {
-                // play sampler with Sync
                 engine.setValue(group, "cue_gotoandplay", 1);
                 engine.setValue(group, "beatsync", 1);
```

Comment:
You removed the comment about playing with sync. Did you mean to remove the syncing?